benchmark.py

```diff
@@ -215,6 +216,8 @@ def raw_bson_func(use_large):
         sys.stdout.flush()
 
         # Test with small and large documents.
+        import faulthandler
+        faulthandler.enable()
```

Comment:
Can you move this to the top of the file so that we always enable the faulthandler?

Reply:
Done.